### PR TITLE
[glog] new recipe

### PR DESCRIPTION
**Pull request by dpo**

### CODE REVIEW

File: G/glog/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,61 @@
+using BinaryBuilder
+
+name = "Glog"
```
Comment (giordano) [inline G/glog/build_tarballs.jl:3]: The directory name must exactly match, capitalisation included, this name.

File: G/glog/build_tarballs.jl [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      -DBUILD_TESTING=OFF \
+      ..
+
+make -j${nproc}
+make install
+
+# Upstream PR from 30 Aug 2017 "Produce pkg-config file under cmake"
+# See https://github.com/google/glog/pull/239 and #483, not released
+mkdir ${libdir}/pkgconfig
+cat >${libdir}/pkgconfig/libglog.pc <<EOS
```
Comment (giordano) [inline G/glog/build_tarballs.jl:30]: The pkg-config file is always installed in `lib/`
```suggestion
mkdir ${prefix}/lib/pkgconfig
cat >${prefix}/lib/pkgconfig/libglog.pc <<EOS
```

File: G/glog/build_tarballs.jl [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+      ..
+
+make -j${nproc}
+make install
+
+# Upstream PR from 30 Aug 2017 "Produce pkg-config file under cmake"
+# See https://github.com/google/glog/pull/239 and #483, not released
+mkdir ${prefix}/lib/pkgconfig
+cat >${prefix}/lib/libglog.pc <<EOS
+  prefix=#{prefix}
```
[the remark this answers is not in the document]
Comment (dpo) [reply G/glog/build_tarballs.jl]: Yes of course, thank you. Copy-pasting from Homebrew formula...

**Comment by dpo** [comment]
Any idea about the build errors on freebsd and windows? I tried adding LibUnwind as dependency on FreeBSD, but no joy.

**Comment by giordano** [comment]
32-bit Windows looks bugged to me.  `cmake` can't find the `UnDecorateSymbolName` symbol:
```
[00:36:46] -- Looking for UnDecorateSymbolName in dbghelp
[00:36:46] -- Looking for UnDecorateSymbolName in dbghelp - not found
```
Yet they don't use the information that the symbol is not there and insist on calling the non-existent function
```
[00:36:51] /opt/bin/i686-w64-mingw32-g++ --sysroot=/opt/i686-w64-mingw32/i686-w64-mingw32/sys-root/    -shared -o libglog.dll -Wl,--out-implib,libglog.dll.a -Wl,--major-image-version,0,--minor-image-version,4 -Wl,--whole-archive CMakeFiles/glog.dir/objects.a -Wl,--no-whole-archive @CMakeFiles/glog.dir/linklibs.rsp
[00:36:51] CMakeFiles/glog.dir/objects.a(demangle.cc.obj):demangle.cc:(.text+0x9e): undefined reference to `_imp__UnDecorateSymbolName@16'
[00:36:51] CMakeFiles/glog.dir/objects.a(symbolize.cc.obj):symbolize.cc:(.text+0x1a3): undefined reference to `_imp__SymFromAddr@20'
[00:36:51] CMakeFiles/glog.dir/objects.a(symbolize.cc.obj):symbolize.cc:(.text$_ZN6google14SymInitializerC1Ev[__ZN6google14SymInitializerC1Ev]+0x2f): undefined reference to `_imp__SymSetOptions@4'
[00:36:51] CMakeFiles/glog.dir/objects.a(symbolize.cc.obj):symbolize.cc:(.text$_ZN6google14SymInitializerC1Ev[__ZN6google14SymInitializerC1Ev]+0x51): undefined reference to `_imp__SymInitialize@12'
[00:36:51] CMakeFiles/glog.dir/objects.a(symbolize.cc.obj):symbolize.cc:(.text$_ZN6google14SymInitializerD1Ev[__ZN6google14SymInitializerD1Ev]+0x12): undefined reference to `_imp__SymCleanup@4'
[00:36:51] collect2: error: ld returned 1 exit status
[00:36:51] make[2]: *** [CMakeFiles/glog.dir/build.make:199: libglog.dll] Error 1
```
I don't see what we can do here, apart from reporting the bug :woman_shrugging:

**Comment by giordano** [comment]
What is even more weird is that the symbol is actually there:
```
sandbox:${WORKSPACE}/srcdir/glog-0.4.0 # nm /opt/i686-w64-mingw32/i686-w64-mingw32/sys-root/lib/libdbghelp.a | grep UnDecorateSymbolName
00000000 T _UnDecorateSymbolNameW@16
00000000 I __imp__UnDecorateSymbolNameW@16
00000000 T _UnDecorateSymbolName@16
00000000 I __imp__UnDecorateSymbolName@16
```

**Comment by dpo** [comment]
Wonderful! Thank you!

**Pull request merged**